src/networkWrapper/wrappers/wsbrowsers.js

```diff
@@ -234,4 +232,25 @@ function poke (listeners, roomId, payload) {
   }
 }
 
+/**
+ * Called when the connection closes with an error state
+ *
+ * @param {boolean} autoReconnect
+ * @param {number} reconnectionDelay
+ * @param {string|Object} message
+ */
+function onClientError(autoReconnect, reconnectionDelay, message) {
```

Comment:
It should be, I separated the two wrappers because I initially thought that there would be more differences than that.

Right now the biggest problem is with webpack, when it tries to package `ws` and other modules that are incompatible with web browsers (crypto and bufferutil).
The only solution I found was to exclude wsnode.js from webpack's analysis. We need to find some way to factorize wsnode and wsbrowsers while preventing webpack to include these modules.